packages/link/src/constants/linkTypes.js

```diff
@@ -0,0 +1,10 @@
+import { fontTypes } from "@crave/farmblocks-theme";
+
+const linkTypes = {
```

Comment:
maybe this file should be named linkColors, since the values on this dict are color hex values in the end of the day

Reply:
I named it this way because of the fontTypes import, which is also a color dictionary, so using it in a 'link' context became 'linkTypes'

Reply:
ok, not a big deal